src/main/scala/com/twilio/guardrail/AbstractGuardrailCodegenMojo.scala

```diff
@@ -48,11 +48,14 @@ abstract class AbstractGuardrailCodegenMojo(phase: Phase) extends AbstractMojo {
   @Parameter(property = "dtoPackage")
   var dtoPackage: String = _
 
+  @Parameter(property = "framework", defaultValue = "akka-http")
+  var framework: String = _
+
   @Parameter(property = "tracing", defaultValue = "false")
   var tracing: Boolean = _
 
-  @Parameter(property = "framework", defaultValue = "akka-http")
-  var framework: String = _
+  @Parameter(required = false, readonly = false)
+  var modules: java.util.List[_] = _
```

Comment:
Why not `java.util.List[String]`?

Reply:
My assumption was that that assertion wouldn't be sound, and I'd get something like:
```
scala> List[Long](1,2,3,4) match { case xs: List[String] => xs: List[String] }
res1: List[String] = List(1L, 2L, 3L, 4L)
```

Reply:
Ah, I see.  I don't think it'd be an issue, but fair to code defensively around it.